packages/app-tests/e2e/fixtures/authentication.ts

```diff
@@ -33,9 +33,63 @@ export const manualLogin = async ({
   }
 };
 
-export const manualSignout = async ({ page }: ManualLoginOptions) => {
+export const manualSignout = async ({ page }: LoginOptions) => {
   await page.waitForTimeout(1000);
   await page.getByTestId('menu-switcher').click();
   await page.getByRole('menuitem', { name: 'Sign Out' }).click();
   await page.waitForURL(`${WEBAPP_BASE_URL}/signin`);
 };
+
+export const apiSignin = async ({
```

Comment:
Why are we doing it this way now?

Reply:
No point testing manual sign in every time we need to access the app, this is faster and more reliable.

The manual sign in is tested in it's own test spec.